src/views/ProfileCreation/Mint.tsx

```diff
@@ -52,8 +49,11 @@ const Mint: React.FC = () => {
       onConfirm: () => {
         return callWithGasPrice(bunnyFactoryContract, 'mintNFT', [variationId])
       },
+      onApproveSuccess: () => {
+        toastSuccess('Enabled', 'Please confirm this NFT for minting')
+      },
       onSuccess: () => {
-        dispatch(fetchWalletNfts(account))
```

Comment:
This is a `/collectibles` state dispatch (that will be removed when the nftmarket is merged) - came in from recent changes on develop. We don't need it any more, we fetch a users' NFTs in the next component (`ProfilePicture`).